src/app/pages/sharing/iscsi/authorizedaccess/authorizedaccess-form/authorizedaccess-form.component.ts

```diff
@@ -171,8 +173,20 @@ export class AuthorizedAccessFormComponent {
             delete errors['manualValidateError'];
             delete errors['manualValidateErrorMsg'];
           }
+          // If the error gets cleared in this comparison validator, make sure it is cleared for both fields
+          if (otherCtrl.hasError('manualValidateError')) {
+            delete otherErrors['manualValidateError'];
+            delete otherErrors['manualValidateErrorMsg'];
+            delete otherErrors['matchesOther'];
```

Comment:
do we need to delete the validation errors of otherCtrl here? since after we reset the value in line183, the validator will check it again.

Reply:
Good point